lms/static/js/views/fields.js

```diff
@@ -74,14 +104,14 @@
             },
 
             title: function (text) {
-                return this.$('.u-field-title').html(text);
+                return HtmlUtils.setHtml(this.$('.u-field-title'), text);
```

Comment:
Actually, if we want to support either HTML or text, we should probably rename `text` to `title`.  If plain text only, then we can switch to what I wrote above.